packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -597,12 +661,63 @@ export class MetriportMedicalApi {
    *
    * @returns True if the signature is verified, false otherwise.
    */
-  verifyWebhookSignature = (wh_key: string, reqBody: string, signature: string): boolean => {
+  verifyWebhookSignature(wh_key: string, reqBody: string, signature: string): boolean {
+    return MetriportMedicalApi.verifyWebhookSignature(wh_key, reqBody, signature);
+  }
```

Comment:
Kept the non-static for backwards compatibility, but also so users who have an instance call call it from the instance (TS doesn't allow to access a static function on an instance).